### DISCUSSION TITLE
generating real time chart

### DISCUSSION THREAD

**Discussion by yoonism49** [katspaugh/wavesurfer.js#2925] [Ideas]
Hi there. Thanks for your great support, first of all. 
Question - Is there a way to generate the chart real time, as the recording is still going on (without stopping)? 

If not, We are trying to  generate the chart every 5 seconds to update the latest noises. Is there a way to get the wave file every five seconds without stopping the player?

**Comment by katspaugh** [comment]
You can probably fork the Record plugin and modify it to render combined chunks every five seconds without stopping the recording.
